user-guide/modules/ROOT/pages/advanced-builds.adoc

```diff
@@ -1,3 +1,247 @@
 = Advanced Builds
+:navtitle: Advanced Builds
```

Comment:
I find it weird to call these "advanced builds"... probably because "advanced" might be a little ambiguous and with the connotation of being "hard", while these are basically just "build options" with nothing particularly hard about them.

Reply:
Agree.

Reply:
Build Scenarios?